src/nihss.ts

```diff
@@ -0,0 +1,26 @@
+type Orientation = {
+    correctMonth: boolean,
+    correctAge: boolean,
+    correctEyes: boolean,
+    correctHand: boolean
+}
+
+function countOrientation(fields: Orientation): number {
+    let score = 0
+    Object.values(fields).forEach(value => {
+        if(!value){
+            score++
+        }
+    })
+    return score
```

Comment:
If you want to use `reduce`, something like this should also work

```suggestion
    return Object.values(orientations).reduce((sum, isSet) => (isSet ? sum : sum + 1), 0)
```

Reply:
Thank you! That will help